lib/wda-mac.js

```diff
@@ -278,19 +284,55 @@ class WDAMacServer {
     }
   }
 
+
+  /**
+   * @typedef {Object} ProxyToInfo
+   *
+   * @property {string} scheme - The scheme proxy to.
+   * @property {string} host - The host name proxy to.
+   * @property {number} port - The port number proxy to.
+   * @property {string} path - The path proxy to.
+   */
+
+  /**
+   * Returns information about proxy information which WDAMacServer proxy to.
+   *
+   * @param {Object} caps - The capabilities provided by a client
+   * @return {ProxyToInfo}
+   */
+  getProxyToInfo (caps) {
+    let scheme = 'http';
+    if (_.isString(caps.webDriverAgentMacUrl)) {
```

Comment:
It would be shorter to flip the condition to `if (!caps.webDriverAgentMacUrl) { return {scheme, host: HOST, port: this.process.port, path: ''} } `